.github/workflows/ci.yml

```diff
@@ -10,38 +10,38 @@ jobs:
     strategy:
       matrix:
         variant:
-          - 7.4
-          - 7.4-alpine
-          - 8.0
+          - "7.4"
+          - "7.4-alpine"
+          - "8.0"
         platform:
         - linux/amd64
-        # - linux/arm64
+        - linux/arm64
         exclude:
-          - props:
-              Dockerfile: Dockerfile-7.4-alpine
+          - variant: 7.4-alpine
           - platform: linux/arm64
     env:
-      TEST_MATCH: PHP Version ${{ matrix.props.version }}
+      TEST_MATCH: PHP Version ${{ matrix.variant }}
     steps:
       -
         name: Checkout
         uses: actions/checkout@v2
+      -
+        name: Install goss
+        run: |
+          curl -fsSL https://goss.rocks/install | sh
```

Comment:
Let's use manual install instead of pipe to sh https://github.com/aelsabbahy/goss#latest